deployment/docker-compose/docker-compose.yml

```diff
@@ -7,30 +7,48 @@ volumes:
 services:
   pyaleph:
     restart: always
-    image: alephim/pyaleph-node:beta
-    command: pyaleph --config /opt/pyaleph/config.yml
+    image: pyaleph:test
+    command: pyaleph --config /opt/pyaleph/config.yml --key-dir /opt/pyaleph/keys -vv
     ports:
-      - 127.0.0.1:8000:8000/tcp
-      - 4024:4024/tcp
-      - 4025:4025/tcp
+      - "127.0.0.1:8000:8000/tcp"
+      - "4024:4024/tcp"
+      - "4025:4025/tcp"
     volumes:
       - ./config.yml:/opt/pyaleph/config.yml
-      - ./node-secret.key:/opt/pyaleph/node-secret.key
+      - ./keys:/opt/pyaleph/keys
     depends_on:
       - mongodb
       - ipfs
+      - p2pd
     networks:
       - pyaleph
     logging:
       options:
         max-size: 50m
 
+  p2pd:
+    restart: always
+    image: jsp2pd:test
+    networks:
+      - pyaleph
+    environment:
+      PRIVATE_KEY_FILE: "/etc/jsp2pd/keys/serialized-node-secret.key"
+      LISTEN_MADDR: "/ip4/0.0.0.0/tcp/4020"
+      HOST_MADDRS: "/ip4/0.0.0.0/tcp/4022"
+      PUBSUB: "true"
+      PUBSUB_ROUTER: "floodsub"
+    ports:
+      - "4020:4020"
+      - "4022:4022"
```

Comment:
Okay I did not take the discovery process into account, it's true that the initial addresses are all in 4025. At runtime the port doesn't matter much, since the node advertises its own address.

I changed the port to make sure I wasn't messing with other config options, it's easy to reverse.